oca_dependencies.txt

```diff
@@ -0,0 +1,5 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+web
+# TODO: change this back to server-tools once https://github.com/OCA/server-tools/pull/738 is merged
```

Comment:
Dependency with: https://github.com/OCA/server-tools/pull/738

My proposal is to first approve PR 738.